examples/sveltekit/package.json

```diff
@@ -12,7 +12,7 @@
   },
   "devDependencies": {
     "@sveltejs/adapter-auto": "next",
-    "@sveltejs/kit": "next",
+    "@sveltejs/kit": "next TODO ",
```

Comment:
`sveltekit` detection (as opposed to `sveltekit-1` detection) looks for a package version that looks like this part of the regex `"1\\.0\\.0-next\\.(\\d+)"`. Should we update the `./examples/sveltekit` example to match?

Reply:
@tcc-sejohnson @leerob what do you think?

Reply:
cc @tcc-sejohnson

Reply:
Done! [`56e2857` (#9197)](https://github.com/vercel/vercel/pull/9197/commits/56e2857bcad8eadbd768a3f04a40234405174aa5)

Reply:
Lets add a lock file if there isn't already one